tensorflow_probability/python/experimental/bijectors/highway_flow.py

```diff
@@ -1,27 +1,29 @@
 import numpy as np
-import tensorflow as tf
-
-import tensorflow_probability as tfp
-
-tfd = tfp.distributions
-tfb = tfp.bijectors
-tfe = tf.experimental
+import tensorflow.compat.v2 as tf
 
+from tensorflow_probability.python import bijectors as tfb
+from tensorflow_probability.python import util
+from tensorflow_probability.python.experimental.bijectors import scalar_function_with_inferred_inverse
 
 def build_highway_flow_layer(width, residual_fraction_initial_value=0.5, activation_fn=None):
     # FIXME: should everything be in float32 or float64?
     # TODO: add control that residual_fraction_initial_value is between 0 and 1
     return HighwayFlow(
         width=width,
-        residual_fraction=tfp.util.TransformedVariable(
+        residual_fraction=util.TransformedVariable(
             initial_value=np.asarray(residual_fraction_initial_value, dtype='float32'),
             bijector=tfb.Sigmoid()),
         activation_fn=activation_fn,
         bias=tf.Variable(np.random.normal(0., 0.01, (width,)), dtype=tf.float32),
```

Comment:
We've been moving in TFP towards using stateless RNGs, which are required by JAX and now also supported by TF, and tend to be easier to reason about (see https://github.com/tensorflow/probability/blob/master/PRNGS.md if you want to learn more). The idiom is along the lines of:

```
# At top
from tensorflow_probability.python.internal import samplers

# Function defn
def build_highway_flow_layer(..., seed=None):
   # Ensure that each part gets a different initialization.
   bias_seed, upper_seed, lower_seed = samplers.split_seed(seed, n=3)
   ...
   # Drawing initial values.
   bias = tf.Variable(samplers.normal([width], mean=0., stddev=0.01, dtype=dtype, seed=bias_seed)),
   upper_diagonal_weights_matrix = TransformedVariable(samplers.normal(..., seed=upper_seed), ...),
   lower_diagonal_weights_matrix = TransformedVariable(samplers.normal(..., seed=lower_seed), ...))
```